tests/benchmark/shared_info.cc

```diff
@@ -0,0 +1,99 @@
+#include "shared_info.h"
+
+extern "C" {
+#include "bin/common.h"
+
+}
+
+struct s2n_cipher_suite *all_suites[] = {
```

Comment:
Can you instead just use `cipher_preferences_test_all` from s2n_cipher_preferences.h? I know we talked about minimizing the use of internal headers/structs but for now I like reducing the duplication and place people will need to update in the future.